Specifications/Events.Processing/mocks.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Logging;
+using Moq;
+
+namespace Dolittle.Runtime.Events
+{
+    public static class mocks
+    {
+        public static Mock<ILogger> a_logger()
```

Comment:
I would just use `Mock.Of<ILogger>()` in the place where it is using logging.

Reply:
Cool. I removed it